.github/workflows/runner-images.yaml

```diff
@@ -0,0 +1,66 @@
+name: runner images
+on:
+  pull_request:
+    paths:
+      - "constants.go"
+      - "runner-images/**"
+      - ".github/workflows/runner-images.yaml"
```

Comment:
I don't think it's appropriate to trigger the Action under these conditions.

For example, if these files are updated but the actions/runner version is not changed, there's a risk of overwriting the image with the same tag.

Ideally, we should only execute this Action when either the controller version or the actions/runner version is updated.